docs/persistence/mq.md

```diff
@@ -0,0 +1,63 @@
+# MQ
+
+Although we recommend using Kafka if possible, MQ is supported on the nais platform. 
+
+
+### Requirements
+
+
+#### Basta
+
+!!! info
+When ordering groups for existing service users the password will be reset
+
+MQ groups and users are ordered using [basta](https://basta.intern.nav.no). 
+For legacy applications registered in [fasit](https://fasit.adeo.no) use "AD groups" on the "Create" tab, 
+for all other applications use "Custom AD groups".
+
+Access to basta (and fasit) can be obtained by requesting access from [identity management](mailto:nav.it.identhandtering@nav.no).
+
+Access to development environments (including legacy u, t and q environments):
+
+``` 0000-GA-env-config-TestAdmin - Fasit T/Q```
```

Comment:
Hvis du skal bruke trippe-fnutt så må du ha linjeskift for at markdown-en skal bli riktig.
```suggestion
```
0000-GA-env-config-TestAdmin - Fasit T/Q
```
```